Scripts/UpdateInsightsTableCreationSettings.ps1

```diff
@@ -0,0 +1,62 @@
+<# 
+.SYNOPSIS 
+Script that enables/disables per process/queue table creation for UiPath Insights.
+#>
+param(
+  [Parameter(Mandatory=$true)][string]$server # Orchestrator endpoint
+  ,[Parameter(Mandatory=$true)][string]$tenancyName # Tenant name
+  ,[Parameter(Mandatory=$true)][string]$user # Tenant user name
+  ,[secureString]$password = $(Read-Host -Prompt "Enter password" -AsSecureString) # Tenant user password (prompted)
+  ,[boolean]$perProcessTablesEnabled = $true # Boolean to enable/disable process tables
+  ,[boolean]$perQueueTablesEnabled = $true # Boolean to enable/disable queue tables
+)
+
+$credentials = New-Object System.Management.Automation.PSCredential `
+  -ArgumentList $user, $password
+
+# Base headers
+$headers = @{
+  'Accept' = 'application/json'
+  'Content-Type' = 'application/json'
+}
+
+$body = "{ 
+  `"tenancyName`": `"$tenancyName`",
+  `"usernameOrEmailAddress`":
+  `"$($credentials.GetNetworkCredential().UserName)`",
+  `"password`": `"$($credentials.GetNetworkCredential().Password)`"
+}"
+
+$targetUri = "$server/api/Account/Authenticate"
+
+# Authenticate with the given tenant.
+$authResult = Invoke-RestMethod -Method Post -Uri $targetUri -Headers $headers -Body $body
+
+if ($null -eq $authResult -or $null -eq $authResult.result) {
+  exit 1
```

Comment:
Write something

Reply:
Since the info is in the exception, went with:

```
PS C:\Workspace\Insights-Customer> .\Scripts\UpdateInsightsTableCreationSettings.ps1 -server 'http://localhost:6235' -tenancyName IT -user admin                                                                                                Enter password:
Invoke-RestMethod : Unable to connect to the remote server
At C:\Workspace\Insights-Customer\Scripts\UpdateInsightsTableCreationSettings.ps1:33 char:15
+ ... uthResult = Invoke-RestMethod -Method Post -Uri $targetUri -Headers $ ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Invoke-RestMethod], WebException
    + FullyQualifiedErrorId : System.Net.WebException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand

Could not authenticate, exiting.
```